README.md

```diff
@@ -55,8 +55,13 @@ Yggdrasil can be configured using a config file e.g:
 {
   "nodeName": "foo",
   "ingressClasses": ["multi-cluster", "multi-cluster-staging"],
-  "cert": "path/to/cert",
-  "key": "path/to/key",
+  "certificates": [
+    {
+      "hosts": ["*.api.com"],
+      "cert": "path/to/cert",
+      "key": "path/to/key"
+    }
+  ],
```

Comment:
probably worth explaining in the readme that these are in the yggdrasil filesystem and beamed over to the envoy nodes